components/blocks/About.vue

```diff
@@ -20,13 +20,19 @@
           {{ item.title }}
         </h2>
         <!-- Opportunities -->
-        <div v-if="item.title === 'Opportunities'" class="card-group">
-          <template v-if="item.data.length > 0">
+        <div
+          v-if="item.title === 'Opportunities' && opportunitiesData.length > 0"
+          class="card-group"
+        >
+          <template v-for="position in opportunitiesData">
             <a
-              v-for="(position, ind) in item.data"
-              :key="'position' + ind"
+              v-if="position.locationsText === '180 George Street'"
```

Comment:
I would do this check on the data query results before putting it in `opportunitiesData` - it will keep the markup cleaner